Assets/MixedRealityToolkit/_Core/Definitions/Utilities/Editor/InvertAxesEnum.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities.Editor
+{
+    /// <summary>
+    /// Used for the editor enum flag field.
+    /// </summary>
+    public enum InvertAxesEnum
+    {
+        Nothing = 0,
+        X = 1,
+        Y = 2,
+        Both = ~0
```

Comment:
probably need to specify both x & y here instead of the bit assignment bc you didn't specify flags attribute. But also Unity is picky about how it'll render it in the inspector.

Reply:
Specifying it as `Both = X & Y` with or without flags yields this

![everything](https://user-images.githubusercontent.com/9560490/44556649-61f3b400-a700-11e8-9740-0adf1313002b.png)

Reply:
Could always define it as 3

Reply:
[EditorGUILayout.EnumFlagsField](https://docs.unity3d.com/ScriptReference/EditorGUILayout.EnumFlagsField.html) says:
>Note: This method only supports enums whose underlying types are supported by Unity's >serialization system (sbyte, short, int, byte, ushort, or uint). For enums backed by an unsigned ?>type, the "Everything" option should have the value corresponding to all bits set (i.e. ~0 in an >unchecked context or the MaxValue constant for the type).

so 

```
[System.Flags]
public enum InvertAxesEnum : uint
{
    Nothing = 0,
    X = 1,
    Y = 2,
    Both = uint.MaxValue
}
```

is valid, same as

```
[System.Flags]
public enum InvertAxesEnum
{
    Nothing = 0,
    X = 1,
    Y = 2,
    Both = ~0
}
```

Reply:
Idk, I usually just just the IntPopup